addons/point_of_sale/static/src/app/models/pos_order.js

```diff
@@ -113,16 +116,16 @@ export class PosOrder extends Base {
             pos_qr_code:
                 this.company.point_of_sale_use_ticket_qr_code &&
                 qrCodeSrc(
-                    `${this.pos.base_url}/pos/ticket/validate?access_token=${this.access_token}`
```

Comment:
hmm, maybe make the export_for_printing accept an argument, like an object that will contain the 'base_url'? I want to use object because if later it needed more, we can just add new item in the object.